src/convergents.jl

```diff
@@ -0,0 +1,22 @@
+immutable ConvergentIterator{T<:Integer,CF<:ContinuedFraction}
```

Comment:
Why does this need a generic parameter `T`? Isn't that parameter implied by `CF`?